index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
```

Comment:
- [ ] Please your Pll Request **_title_** and **Summary** are not professionally done. Kindly edit them to reflect this project. Eg. 
- Title: Hello Microverse Project
- Description: In this pull request, I added the following: list the things you did.

Learn how you can change the title - https://www.youtube.com/watch?v=cnIDkME-gg0

- [ ] The branch should have a name of a feature or action (e.g. 'created_sidebar' or 'added_comments_section') and not `features`. To fix this. please checkout to a new branch and use the correct name and mention the old PR in the new PR.

![image](https://user-images.githubusercontent.com/84156714/218822840-bd150834-cc03-49f6-8294-7340d980c17e.png)